custom/enikshay/reports/filters.py

```diff
@@ -0,0 +1,34 @@
+from django.core.urlresolvers import reverse
+
+from corehq.apps.locations.models import SQLLocation
+from corehq.apps.reports.filters.base import BaseMultipleOptionFilter
+
+
+class EnikshayLocationFilter(BaseMultipleOptionFilter):
```

Comment:
Yeah, would be good to make this inherit from `LocationChoiceProvider` so that it also gets the location restriction features I'm currently working on (which are required for these reports)